website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -382,6 +382,20 @@ The `policy` field for the plugin rule can have one of the following values:
 - `deny`: do not allow the resource to be read or modified. Deny takes
   precedence when multiple policies are associated with a token.
 
+## UI rules
+
+Since the Nomad UI provides a much richer user experience than the CLI, additional API endpoints `/v1/agent` `/v1/node` are used when rendering pages.  Adding the rules below to your ACL when utilizing the Nomad UI is recommended.
```

Comment:
Generally we want to keep marketing language out of the docs and have it be more "just the facts". Also, we hard-wrap markdown:

```suggestion
The Nomad web UI uses the API endpoints `/v1/agent` and `/v1/node` for nearly
every page. ACL policies for users who will use the Nomad UI should include the
following rules.
```